packages/system/src/components/button.css.ts

```diff
@@ -16,7 +16,7 @@ const buttonVariants = {
 			sprinkles({
 				paddingX: 24,
 			}),
-			{ height: pxToRem(48) },
+			{ height: pxToRem(48) , width : "92%"},
```

Comment:
Remove these styles and only edit the button styles in the `contributor-banner` file. 
You want to avoid making changes to a button component where it is used in multiple places. 
It is always best to make style changes to the file you are working with. In this case, that file would be the `contributor-banner` file.

Right now, this changes actually breaks styles in other places like here
<img width="306" alt="Screen Shot 2022-12-12 at 4 17 57 PM" src="https://user-images.githubusercontent.com/67210629/207193883-8b5d40ac-9444-44f0-96c5-a924b360ed1e.png">